packages/firestore/test/unit/model/mutation.test.ts

```diff
@@ -270,12 +270,13 @@ describe('Mutation', () => {
   // we don't have unit tests for it currently. We could consider removing this
   // test once we have integration tests.
   it('can create arrayRemove() transform.', () => {
-    const transform = transformMutation('collection/key', {
+    const transform = patchMutation('collection/key', {
       foo: FieldValue.arrayRemove('tag')
     });
-    expect(transform.fieldTransforms.length).to.equal(1);
+    expect(transform.fieldTransforms).to.not.be.undefined;
```

Comment:
This should always be true, shouldn't it?

Reply:
Removed here and throughout.